relay-general/src/pii/config.rs

```diff
@@ -42,11 +65,8 @@ impl Serialize for Pattern {
 impl<'de> Deserialize<'de> for Pattern {
     fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
         let raw = String::deserialize(deserializer)?;
-        let pattern = RegexBuilder::new(&raw)
-            .size_limit(262_144)
-            .build()
-            .map_err(Error::custom)?;
-        Ok(Pattern(pattern))
+        let pattern = Pattern::new(&raw, false).map_err(Error::custom)?;
+        Ok(pattern)
```

Comment:
This should work now:

```suggestion
        Pattern::new(&raw, false).map_err(Error::custom)
```